### DISCUSSION TITLE
Vue 2 compatibility?

### DISCUSSION THREAD

**Discussion by nicooprat** [vuejs/vitepress#432] [General]
Could find anything about compatibility with Vue 2. We're currently moving our app from `vue-cli` to `vite`. I see that Vue 3 is specifically needed in the [`package.json`](https://github.com/vuejs/vitepress/blob/main/package.json#L77) but is there anything technically that would make it incompatible with Vue 2?

I see in the docs that Vue 3 is better about static optimization, but this could be a feature more than a requirement: https://vitepress.vuejs.org/#it-uses-vue-3

Thanks.

**Comment by ckvv** [comment]
Is this what you are looking for <https://github.com/vitejs/awesome-vite#vue-2>

**Comment by nicooprat** [comment]
I was looking for something more specific to Vitepress, not only Vite. Did I miss something?

**Comment by yyx990803** [comment]
VitePress is Vue 3 only. There is no plan to be Vue 2 compatible.
